test/schema-parsing.js

```diff
@@ -596,6 +596,14 @@ describe('Schema parsing', () => {
             _attributes: {
               required: false
             }
+          },
+          postalCode: {
+            _attributes: {
+              name: 'address.postalCode',
```

Comment:
@joeosburn `name` should be getting set dynamic. Did it not?

Reply:
I believe it is being named dynamically, and correctly, because the field is named address here:
https://github.com/shanebo/formgive/blob/bf4c601a028dd69e9d170696ae361060739029ed/test/schema-parsing.js#L591

Or am I misunderstanding something?